MVVM Module/Source/Feed Presentation/FeedViewModel.swift

```
@@ -20,14 +20,20 @@ final class FeedViewModel {
 
 	var onLoadingStateChange: Observer<Bool>?
 	var onFeedLoad: Observer<[FeedImage]>?
+	var onError: Observer<String?>?
```

Suggestion: `onError` could be renamed `onErrorStateChange`, since we use this property also when no error has occurred, to hide the error view.

---

you are right. Will change that.